nb/projects/micro_gear_motor/micro_gear_motor.md

```diff
@@ -172,9 +172,9 @@ Motor Driver Holder
 
 ```JavaScript
 const wemosDriverHolder = Block(4, 5, 3.2 * 11)
-  .cut({ mode: 'basic' }, Box(25.7 + 0.2, 34.5 + 0.2).ez(2, 100))
```

Comment:
Are we operating in a single mode now?

Reply:
Yeah, we might need this back, but let's finish consolidating first.